application/controllers/handler.js

```diff
@@ -40,9 +40,14 @@ module.exports = {
         console.log('identity already taken: ', user.email, user.username, result);
         if (result.assigned === false) {
           //Send email before creating the user
+          let message = 'Dear '+user.forename+' '+user.surname+',\n\nwelcome to SlideWiki! You have registered your account with the username '+user.username+'. In order to activate your account please use the following link:\n\n https://'+req.info.host+'/user/activate/'+user.email+'/'+user.activate_secret+'\n\nGreetings,\nthe SlideWiki Team';
+          if (!config.SMTP.enabled) {
```

Comment:
You're defining an email message in order to send an email in case SMTP is disabled?

Reply:
Right. I removed that message is changed when SMTP is disabled. I leave the message in the variable because its a long line.